mff_rams_plugin/templates/static_views/dealerRules.html

```diff
@@ -0,0 +1,34 @@
+<ul>
+<li>Your displays and products cannot block the aisles or walkways, but must remain within the table or booth space purchased.</li>
+<li>No selling out of a hotel room, away from the dealer area, or away from your table.</li>
+<li>A list of confirmed dealers will be posted to the web site and will be included in our ConBook. This listing will include your business name and the description of your merchandise from your application, along with a link to your website if one was listed on your dealer application. Dealers are encouraged to check their listing on the web site and promptly request any changes or corrections.</li>
+<li>Advertising space is available in our convention book. If you are interested in purchasing an ad please contact {{ c.PUBLICATIONS_EMAIL }}.</li>
```

Comment:
Adding variables like this that aren't defined anywhere is generally bad practice, because other events (or in this case, future-you) may forget that you need this in your local config file and break the system.

That's not enough to hold up this PR, but now or soon in the future you'll want to add this and other custom variables to this plugin's configspec.ini.